SmartDeviceLink-iOS/SmartDeviceLink/SDLStreamingMediaManager.m

```diff
@@ -229,83 +251,52 @@ - (BOOL)sdl_configureVideoEncoderWithError:(NSError *__autoreleasing *)error {
 
     if (status != noErr) {
         // TODO: Log the error
-        if (*error != nil) {
+        if (*error == nil) {
             *error = [NSError errorWithDomain:SDLErrorDomainStreamingMediaVideo code:SDLStreamingVideoErrorConfigurationCompressionSessionCreationFailure userInfo:@{ @"OSStatus" : @(status) }];
         }
 
         return NO;
     }
-
-    // Set the bitrate of our video compression
-    int bitRate = 5000;
-    CFNumberRef bitRateNumRef = CFNumberCreate(NULL, kCFNumberSInt32Type, &bitRate);
-    if (bitRateNumRef == NULL) {
-        // TODO: Log & end session
-        if (*error != nil) {
-            *error = [NSError errorWithDomain:SDLErrorDomainStreamingMediaVideo code:SDLStreamingVideoErrorConfigurationAllocationFailure userInfo:nil];
-        }
-
-        return NO;
+    
+    if (self.videoEncoderSettings == nil) {
```

Comment:
initWithEncoderSettings would collide with other fixes coming down the pipeline (such as dynamic encoder dimensions). The thought here is that so long as the stream isn't open, we can modify the existing `SDLStreamingMediaManager` object, instead of having to specify it via the proxy (since the developer never calls the init). We can definitely add it in the init.